scripts/serve-local.sh

```diff
@@ -86,5 +86,6 @@ else
   # Only run containers for db and assets
   docker-compose -f docker-compose.yml -f docker-compose.local_override.yml up -d db assets
 
+  sh $CURRENT_DIR/../scripts/setup-undockered-data.sh
```

Comment:
Could we have a comment here about why this script is called here? Looks like the other script is running existing migrations locally and then also loading data, irregardless of the other flags you set up fro serve-local. I'm not necessarily opposed to this, but I just want to understand the what and why.

Reply:
Undockerized Joplin doesn't have an entrypoint, so we need a script to run migrations and seed data. I updated setup-undockered-data to only load data if LOAD_DATA=on is passed, so it conforms to our existing flag setup.